packages/shared/catalogs/sagas.ts

```diff
@@ -54,9 +63,39 @@ const BASE_BUILDER_DOWNLOAD_URL = `${BUILDER_SERVER_URL}/storage/contents`
  *
  */
 export function* catalogsSaga(): any {
-  yield takeEvery(WEARABLES_REQUEST, handleWearablesRequest)
+  yield takeEvery(WEARABLES_REQUEST, handleItemRequest)
   yield takeEvery(WEARABLES_SUCCESS, handleWearablesSuccess)
   yield takeEvery(WEARABLES_FAILURE, handleWearablesFailure)
+  yield takeEvery(EMOTES_REQUEST, handleItemRequest)
+  yield takeEvery(EMOTES_SUCCESS, handleWearablesSuccess)
+  yield takeEvery(EMOTES_FAILURE, handleWearablesFailure)
```

Comment:
if the request handler was renamed from `wearable` to `item` should the success and failure handlers be renamed as well?

Reply:
absolutely, updated! :)